### PR TITLE
Standardize LO's across modules

### PR DESCRIPTION
**Pull request by joelostblom**
@flor14 Sorry that it has taking me so long to get the revision PRs going. Here is the first one with some LO formatting and a suggestion for what the course LOs could be (based on one from each chapter). The next PR with mod0 and mod1 will be up soon.

### CODE REVIEW

**Comment by flor14** [comment]
Don't worry! I will have a look later today.

File: chapters/en/slides/module2/module2_00.md [hunk 1]
```diff
@@ -6,17 +6,19 @@ type: slides
 
 ---
 
+## Module Learning Outcomes
+
 In this module you will learn how to
 
-1. Use Bash to navigate the computer’s filesystem
+1. Use the shell to navigate the computer’s filesystem
 2. Creating new files and directories
```
Comment (flor14) [inline chapters/en/slides/module2/module2_00.md:14]:
```suggestion
2. Create new files and directories
```

File: chapters/en/slides/module2/module2_end.md [hunk 2]
```diff
@@ -4,32 +4,22 @@ type: slides
...
 ---
 
-# Module Learning Outcomes
+## Module Learning Outcomes
 
 In this module you learned how to
 
-1. Use Bash to navigate the computer’s filesystem
+1. Use the shell to navigate the computer’s filesystem
 2. Creating new files and directories
```
Comment (flor14) [inline chapters/en/slides/module2/module2_end.md:14]:
```suggestion
2. Create new files and directories
```

File: chapters/en/slides/module7/module7_00.md [hunk 3]
```diff
@@ -8,21 +8,23 @@ type: slides
...
 ## Module Learning Outcomes
 
-<br>
-
-Hi, and welcome to Module 7 of this course 👋
+By the end of this module, you will be able to
 
-<br>
+1. Explain what JupyterLab is.
+2. Install and use JupyterLab.
```
Comment (flor14) [inline chapters/en/slides/module7/module7_00.md:14]: Installing the software is not one of the learning objectives of module 0?

File: chapters/en/slides/module8/module8_00.md [hunk 4]
```diff
@@ -10,22 +10,20 @@ type: slides
 
 By the end of this module, you will be able to
 
-- Explain the Jupyter Book ecosystem
-- Install Jupyter Book
-- Build your book
-- Create your own content files
-- Publish your book online
+1. Explain the Jupyter Book ecosystem.
+2. Install Jupyter Book.
```
Comment (flor14) [inline chapters/en/slides/module8/module8_00.md:14]: Again, is this not already installed at this point?

**Review by flor14 — COMMENTED** [review]
Done @joelostblom!

File: chapters/en/slides/module7/module7_00.md (hunk 3 above)
Comment (joelostblom) [reply chapters/en/slides/module7/module7_00.md]: Yes you are correct, it is still in the slides, but we don't need to highlight it as a learning objective. I will add a note about conda environments instead: "Work with Conda virtual environments inside JupyterLab"

File: chapters/en/slides/module8/module8_00.md (hunk 4 above)
Comment (joelostblom) [reply chapters/en/slides/module8/module8_00.md]: Yup it is, I removed this LO and will clarify in the slide that it is repetition when we talk about installation.

**Review by joelostblom — COMMENTED** [review]
Thanks for the quick review @flor14! I included your suggestions and will merge now.

**Pull request merged**